src/OpenSage.Game/Data/Apt/Characters/Button.cs

```diff
@@ -44,7 +44,8 @@ public struct ButtonRecord
         public static ButtonRecord Parse(BinaryReader reader)
         {
             return new ButtonRecord(
-                reader.ReadUInt32AsEnumFlags<ButtonRecordFlags>(),
+                reader.ReadByteAsEnumFlags<ButtonRecordFlags>(),
```

Comment:
In the binary file, is the enum actually stored in a uint32? If so, I think this was better before this change.

Reply:
Couldn't do it like before, because i encountered some other bits in the uint24. I don't know what those mean yet, but it makes the Enum verification fail